usama/README.md

```diff
@@ -0,0 +1,59 @@
+#  Web Health Monitoring Lambda Function
```

Comment:
- Beautifully arranged, all the commands. 
- I like the way you've explained to activate the virtual environment for both MAC and Windows users. 
Keep it up:)